alignak_module_backend/arbiter/module.py

```diff
@@ -685,30 +726,59 @@ def hook_tick(self, arbiter):
         :type arbiter: object
         :return: None
         """
-        if int(time.time()) > self.next_check:
-            logger.debug('Check if config in backend has changed')
-            resources = ['realm', 'command', 'timeperiod', 'usergroup', 'user', 'hostgroup',
-                         'host', 'servicegroup', 'service', 'hostdependency',
-                         'hostescalation', 'servicedependency', 'serviceescalation',
-                         'trigger']
-            reload_conf = False
-            for resource in resources:
-                ret = self.backend.get(resource, {'where': '{"_updated":{"$gte": "' +
-                                                           self.time_loaded_conf + '"}}'})
-                if ret['_meta']['total'] > -1:
-                    reload_conf = True
-            if reload_conf:
-                logger.warning('Hey, we must reload conf from backend !!!!')
-                with open(arbiter.pidfile, 'r') as f:
-                    arbiterpid = f.readline()
-                os.kill(int(arbiterpid), signal.SIGHUP)
-            self.next_check = int(time.time()) + (60 * self.verify_modification)
-
-        if int(time.time()) > self.next_action_check:
-            self.get_acknowledge(arbiter)
-            self.get_downtime(arbiter)
-            self.get_forcecheck(arbiter)
-            self.next_action_check = int(time.time()) + self.action_check
+        try:
+            now = int(time.time())
+            if now > self.next_check:
+                logger.info(
+                    "[Backend Arbiter] Check if system configuration changed in the backend..."
+                )
+                resources = [
+                    'realm', 'command', 'timeperiod',
+                    'usergroup',
+                    # 'user',
+                    'hostgroup', 'host', 'hostdependency', 'hostescalation',
+                    'servicegroup', 'service', 'servicedependency', 'serviceescalation',
+                    'trigger'
+                ]
+                reload_conf = False
+                for resource in resources:
+                    ret = self.backend.get(resource, {'where': '{"_updated":{"$gte": "' +
+                                                               self.time_loaded_conf + '"}}'})
+                    logger.info(
+                        " - backend updated resource: %s, count: %d",
+                        resource, ret['_meta']['total']
+                    )
+                    if ret['_meta']['total'] > 0:
+                        reload_conf = True
+                if reload_conf:
+                    logger.warning(
+                        "[Backend Arbiter] Hey, we must reload configuration from the backend!"
```

Comment:
Why this choice?
 In the current development step, having information about data that changed is interesting and having a warning when the backend configuration is reloaded semt interesting too ...